js/main.js

```diff
@@ -1,32 +1,20 @@
 /* global data */
 /* exported data */
-
 var imgInput = document.getElementById('imageInput');
 var imgOutput = document.getElementById('imageOutput');
 var form = document.querySelector('form');
 
-var data = {
-  view: 'entry-form',
-  entries: [],
-  editing: null,
-  nextEntryId: 1
-};
-
-var storage = localStorage.getItem('code-journal');
-if (storage !== null) {
-  data = JSON.parse(storage);
-}
-
 imgInput.addEventListener('blur', function (event) {
   var imgURL = imgInput.value;
   imgOutput.src = imgURL;
 });
 
 form.addEventListener('submit', function (event) {
   event.preventDefault();
-  var title = event.currentTarget[0].value;
-  var photoUrl = event.currentTarget[1].value;
-  var notes = event.currentTarget[2].value;
+
+  var title = form.elements[0].value;
+  var photoUrl = form.elements[1].value;
+  var notes = form.elements[2].value;
```

Comment:
Instead of using indexes, use the name of the input. For instance, `form.elements.title.value`